pkg/security/tpm.go

```diff
@@ -0,0 +1,48 @@
+package security
+
+import (
+	"crypto/sha1"
+
+	tspi "github.com/google/go-tpm/tpm"
+)
+
+const (
+	tpmDefaultDevice string = "/dev/tpm0"
```

Comment:
this might be a public var instead of a private const - will make testing easier

The methods below don't accept a device name, so I would rather rename this to TPMDevice (or TpmDevice).

If you accept the other changes I suggested below, I would leave it to `TpmDefaultDevice` (because the suggested approach would let you override the device name via struct fields)

Reply:
Done